FiligranHQ/xtm-hub
# [backend-frontend] - rework document deletion #71

# Context 

Add securities on document deletion on partner Vault. 

This is part of the #71 : (last two checkpoints) 

# How to test : 

On local, on a partner Vault as an admin orga, click on delete a document. You should see a confirmation popup : 
![image](https://github.com/user-attachments/assets/7ed96171-1e7c-4a50-bde6-a020f3da384c)
 
 Then, on click delete : 
 
![image](https://github.com/user-attachments/assets/61ef21c6-5885-4a4b-996a-a8a4b8883641)

The document should be inactive and the remove_id should be set
